.github/workflows/build-deploy-api-v2.yaml

```diff
@@ -167,15 +167,31 @@ jobs:
     environment: API-Dev
     needs: [ DEV-cleanup ]
     steps:
+      - name: Message Teams Channel on Success
+        run: |
+          curl --location --request POST ${{ secrets.TEAMS_SUCCESS_URL }} \
+          --header 'Content-Type: text/plain' \
+          --data-raw '{
+            "@context": "https://schema.org/extensions",
+            "@type": "MessageCard",
+            "themeColor": "0072C6",
+            "title": "SUCCESS! Deployment to PIA ${{ secrets.ENVIRONMENT }} Environment has finished.",
```

Comment:
if possible, the title should be the name of the ticket and git commit message header.  For e.g 
[DEV] - UTOPIA-737: Adding Message to Teams Channel on Deploy Failure 

Right now , they all look the same. It is hard to pay attention to these messages. The green icon on the left is sufficient to indicate Success. 
![image](https://user-images.githubusercontent.com/8980101/216111634-cb68bd0d-598f-4655-b7ba-224742b4a51f.png)

Reply:
ok. Not sure how about the internal workings.  Will take your word :)